### DISCUSSION TITLE
Connecting Teable to Dbeaver

### DISCUSSION THREAD

**Discussion by vasnt** [teableio/teable#529] [Q&A]
How to connect teable database to dbeaver?

**Comment by christophberger** [comment]
If you run the docker version, you can pick the connection parameters (DB name, user, password,...) from the `.env` file in `dockers/examples/<example_name>`. Ensure to set the database port to the port number in PUBLIC_DATABASE_PROXY. The real database port is hidden inside the container and mapped to that port.
